gui.py

```diff
@@ -69,34 +69,37 @@ def setTextareas(self):
         self.grid.addWidget(self.outText, 0, 1)
 
     def addConvertedRow(self, i):
-        hb = QGridLayout()
+        hb = QHBoxLayout()
         label = QLabel()
         text = self.binfile.get_read_file_name_by_index(i)
         label.setText(text)
-        hb.addWidget(label, 0, 0, 1, 2)
+        label.setFixedWidth(260)
+        hb.addWidget(label)
         saveMe = QPushButton("Save")
         saveMe.clicked.connect(lambda: self.saveAFileFromMany(i))
-        hb.addWidget(saveMe, 0, 2, 1, 2)
+        hb.addWidget(saveMe)
```

Comment:
Here was a confilict, could you confirm that it works correctly @PiotrDobras @KamilMostowski1997 ?

Reply:
It works, the layout was changed from Grid to HBox, that's why "addWidget" command changed too